test/routes/signup.js

```diff
@@ -0,0 +1,127 @@
+var request = require('supertest'),
+    should = require('should'),
+    mongoose = require('mongoose'),
+    config = require('../../config/env/all');
+
+describe('Signup Integration test', function () {
```

Comment:
@andela-akolawole 
Your Test should have an After Block that work on cleaning the DB after each describe block.